### ISSUE TITLE
Migration dependency non-existent

### ISSUE THREAD

**Issue by GuilleAmutio** [neon-jungle/wagtail-birdsong#34]
Hello!

I'm quite new to Wagtail but I'm trying to install Birdsong and last week it was working with no issues but seems like a new update is not executing a migration

`django.db.migrations.exceptions.NodeNotFoundError: Migration birdsong.0007_alter_contacttag_tag dependencies reference nonexistent parent node ('taggit', '0005_auto_20220424_2025')`

I don´t really know if thats something I did wrong or is related with the latest change.

**Comment by seb-b** [comment]
yeah I think a bad migration was accidentally added, the dependency might just need to be updated, or maybe just clean it out

In the meantime I think you can get around this by running `django-admin migrate --fake birdsong 0007`
